edx/edx-enterprise
# ENT-140: Allow catalog to be disabled for EnterpriseCustomer

**Description:**

This pull request adds a migration and code to ensure that when saving an EnterpriseCustomerAdminForm, a user may select a catalog, save the object, and then later remove that catalog from being linked to the EnterpriseCustomer.

**JIRA:** [ENT-140](https://openedx.atlassian.net/browse/ENT-140)

**Dependencies:** None

**Merge deadline:** 30 January 2017

**Testing instructions:**

1. With the existing code, and with a working Course Discovery devstack, create an enterprise customer, and select a catalog from the Course Catalog dropdown menu.
2. Save the enterprise customer
3. Attempt to switch the enterprise customer back to using no catalog
4. Save the enterprise customer
5. Observe that the enterprise customer keeps its existing catalog
6. Switch to this code.
7. Attempt to switch the enterprise customer back to using no catalog
8. Save the enterprise customer
9. Observe that the change has been saved.

**Reviewers:**

**Merge checklist:**

**Post merge:**
      finished.
